nucypher/control/controllers.py

```diff
@@ -265,15 +266,36 @@ def make_control_transport(self):
         # Return FlaskApp decorator
         return self._transport
 
-    def start(self, http_port: int, dry_run: bool = False):
+    def start(self,
+              port: int,
+              tls_key_filepath: str = None,
+              certificate_filepath: str = None,
+              dry_run: bool = False):
 
-        self.log.info("Starting HTTP Control...")
+        self.log.info("Starting HTTP/HTTPS Control...")
```

Comment:
Might as well move the log lower since you know whether or not the process is being launched with HTTPS or not.

Reply:
✔️